### PR TITLE
Add robot.txt

### PR DESCRIPTION
**Pull request by WayneLee97**
which will be changed later. And I put it in the master branch is because this is a global file and should be updated all the time. just remember to come here every time you made some changes in other branches.

### CODE REVIEW

**Comment by EgMan** [comment]
Commit title should be present progressive action like "Add robot.txt."  The rest can be in the commit body.

**Review by ElementMo — APPROVED** [review]

File: robot.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+User-agent:*
+Disallow:
```
Comment (EgMan) [inline robot.txt]: We should probably Disallow crawling our source code.

Comment (WayneLee97) [reply robot.txt]: update the file

**Comment by WayneLee97** [comment]
updated the commit message

**Review by EgMan — APPROVED** [review]

**Review by ElementMo — APPROVED** [review]
Good work!

**Review by user833 — COMMENTED** [review]
that's good

**Pull request merged**